prisma/migrations/20210529200211_add_tables_for_nextjs_auth/migration.sql

```diff
@@ -0,0 +1,77 @@
+-- CreateTable
+CREATE TABLE "Account" (
+    "id" TEXT NOT NULL,
+    "userId" TEXT NOT NULL,
+    "providerType" TEXT NOT NULL,
+    "providerId" TEXT NOT NULL,
+    "providerAccountId" TEXT NOT NULL,
+    "refreshToken" TEXT,
+    "accessToken" TEXT,
+    "accessTokenExpires" TIMESTAMP(3),
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
For anything time stamp, I think we should use TIMESTAMPTZ https://medium.com/building-the-system/how-to-store-dates-and-times-in-postgresql-269bda8d6403

Reply:
This is generated by prisma - if you would like to change it I will keep that in mind for next time!